Source/BSN.Commons/Infrastructure/MessageBroker/RabbitMqEventAggregator/BrokerType.cs

```diff
@@ -8,21 +8,21 @@ public enum BrokerType
         /// <summary>
         /// Direct exchange type routes messages based on exact matching of routing keys.
         /// </summary>
-        Direct,
-        
+        direct,
```

Comment:
Why?

Our convention is Camel Case

Reply:
in https://github.com/Fr-Soltanzadeh/Commons/blob/RabbitMq-bug-fix/Source/BSN.Commons/Infrastructure/MessageBroker/RabbitMqEventAggregator/RabbitMqEventAggregator.cs
line channel.ExchangeDeclare(exchangeOptions.BrokerName, exchangeOptions.ExchangeType.ToString(), ...
we got error that exchange type can not be Direct , 

it must be direct to be ok.

Reply:
Is it ok to use exchangeOptions.ExchangeType.ToString().ToLower() instead of changes I had made?

Reply:
Yes

Reply:
In our code style, Enum starts with Capital

Reply:
OK. Done.